service/lib/agama/storage/iscsi/adapter.rb

```diff
@@ -67,8 +67,7 @@ def read_initiator
           Initiator.new.tap do |initiator|
             initiator.name = Yast::IscsiClientLib.initiatorname
             initiator.offload_card = Yast::IscsiClientLib.GetOffloadCard()
```

Comment:
Could the `#offloard_card` be removed from the `Initiator` class?

Reply:
Removed.